tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -435,6 +438,31 @@ can replace the above code with a single line::
 
     @onready var my_label = get_node("MyLabel")
 
+.. warning::
+
+    Applying ``@onready`` and any ``@export`` annotation to the same variable
+    doesn't work as you might expect. The ``@onready`` annotation will make
```

Comment:
```suggestion
    doesn't work as you might expect. The ``@onready`` annotation will cause
```

Reply:
I copied the text from here:

![](https://github.com/godotengine/godot-docs/assets/47700418/c7f73594-4f99-4884-a632-20fafcff9d53)